crates/bevy_tasks/src/task_pool.rs

```diff
@@ -309,6 +336,42 @@ mod tests {
         assert_eq!(count.load(Ordering::Relaxed), 100);
     }
 
+    #[test]
+    fn test_spawn_pollable() {
+        let transform_fn = |i: usize| i * 3;
+
+        let pool = TaskPool::new();
```

Comment:
Ideally we'd have some way to limit the number of threads this spawns, since each task is extremely lightweight. That's not a problem for this PR to solve